mygraph.py

```diff
@@ -208,34 +226,48 @@ def __init__(self, directed: bool, n: int=0, simple: bool=False):
         self._e = list()
         self._simple = simple
         self._directed = directed
-        self._next_label_value = 0
+        self._tag = None
 
```

Comment:
Ik zie het ff niet. Is dit omdat je next_label_value hierboven al gebruikt heb in een andere context?

Reply:
Inderdaad. `_next_label_value` was een attribute van elk `Graph` object, maar dat maakte het doen van het volgende een bitch.

```python
g = Graph(n=1)
g.vertices[0].label = 'spam'
h = Graph()
h.add_vertex(Vertex(label='spam'))
assert g == h  # Hier verwacht ik True, maar is False, omdat `g._next_label_value = 1` en `h._next_label_value = 0`
```

Bovendien vind ik het onlogisch dat een graafobject z'n eigen labels genereert. Dat zou ik juist globaal laten doen, zodat je unieke labels krijgt. Dit heb ik opgelost door het genereren van labels een `@staticmethod` van `Graph` te maken. Achteraf gezien had dit prima helemaal losgetrokken kunnen worden van de `Graph` klasse.